app/renderer/components/Inspector/Inspector.js

```diff
@@ -163,6 +163,7 @@ export default class Inspector extends Component {
             </Card>
           </TabPane>
         </Tabs>
+        <div>FUDGE!</div>
```

Comment:
seems strange to commit this, at least without a comment explaining what it's doing there

Reply:
:cringe:

Reply:
I was doing some hacking to test another feature...

My word choice could have been much worse 😌